framework/application/android_window.cpp

```diff
@@ -111,6 +111,12 @@ bool AndroidWindow::GetNativeHandle(HandleType type, void** handle)
     }
 }
 
+const std::string& AndroidWindow::GetWsiExtension() const
+{
+    static std::string sWsiExtension { VK_KHR_ANDROID_SURFACE_EXTENSION_NAME };
+    return sWsiExtension;
```

Comment:
Can this just return `VK_KHR_ANDROID_SURFACE_EXTENSION_NAME` and let the return construct the std::string?

Reply:
done